lib/oper.g

```diff
@@ -1931,6 +1931,36 @@ end);
 
 fi;
 
+if BASE_SIZE_METHODS_OPER_ENTRY <> 5 then
+    Error("MethodsOperation must be updated for new BASE_SIZE_METHODS_OPER_ENTRY");
+fi;
+
+# TODO: document this?!
+BIND_GLOBAL("MethodsOperation", function(oper, nargs)
+    local meths, len, result, i, m;
+
+    meths := METHODS_OPERATION(oper, nargs);
+    if meths = fail then
+        return fail;
+    fi;
+    len := BASE_SIZE_METHODS_OPER_ENTRY + nargs;
+    result := [];
+    for i in [0, len .. LENGTH(meths) - len] do
+        m := rec(
+            famPred := meths[i + 1],
+            argFilt := meths{[i + 2 .. i + nargs + 1]},
```

Comment:
Anybody want to nitpick the record names chosen here? E.g. `argFilt` could also be `filters` or `argumentFilters`. In the documentation for `InstallMethod`, it is called `args-filts` (which isn't a valid identifier, though).  And `famPred` is `famp`, `rank` is `val`, and `func` is `method`.

Otherwise, I'd say we can merge this, no?

Reply:
Yes, I think this should be merged.